src/store/RelayStoreData.js

```diff
@@ -632,6 +633,14 @@ class RelayStoreData {
       nodeRangeMap,
     } = obj;
 
+    for (const key in records) {
+      const record = records[key];
+      const range = record.__range__;
+      if (range) {
+        record.__range__ = GraphQLRange.fromJSON(range);
+      }
+    }
+
```

Comment:
Since it would only be used locally to the module, I would add it as a helper at the bottom of the file